dojo/tools/spotbugs/parser.py

```diff
@@ -4,9 +4,8 @@
 __status__ = "Development"
 
 import re
-
+from bs4 import BeautifulSoup
```

Comment:
We use `html2text` for this part of the project. Please switch to the package `html2text`

Reply:
Done :)